subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -843,6 +847,65 @@ static int cmd_wifi_reg_domain(const struct shell *sh, size_t argc,
 	return 0;
 }
 
+static int cmd_wifi_listen_interval(const struct shell *sh, size_t argc, char *argv[])
+{
+	struct net_if *iface = net_if_get_default();
+	struct wifi_ps_params params = { 0 };
+	long interval = 10;
+	int err = 0;
+
+	context.sh = sh;
+
+	if (!parse_number(sh, &interval, argv[1], 3, 65535)) {
+		return -EINVAL;
+	}
+
+	params.listen_interval = interval;
+	params.ps_flag = WIFI_PS_PARAM_LISTEN_INTERVAL;
+
+	if (net_mgmt(NET_REQUEST_WIFI_LISTEN_INTERVAL, iface, &params, sizeof(params))) {
```

Comment:
`NET_REQUEST_WIFI_LISTEN_INTERVAL` where is this defined, I cannot seem to find it anywhere?